src/components/ebay-tourtip/index.js

```diff
@@ -0,0 +1,39 @@
+const processHtmlAttributes = require('../../common/html-attributes');
+const emitAndFire = require('../../common/emit-and-fire');
+const template = require('./template.marko');
+
+function getInitialState(input) {
+    input.location = input.location || 'bottom';
+    input.htmlAttributes = processHtmlAttributes(input);
+    input.hostSelector = '.tourtip__host';
+    input.overlaySelector = '.tourtip__overlay';
+    input.expanded = true;
+
+    return input;
+}
+
+function getTemplateData(state) {
+    return state;
+}
+
+function init() {
+    this.setState('expandInit', true);
```

Comment:
What is the purpose of waiting for the component to mount before making it expanded?

Reply:
It's because it's location is wrong and jumps after it's loaded.

Reply:
You know, I think this is mitigated by my refactor. It appears I don't need it anymore. I'll look at this in the morning.